kubernetes/usage.md

```
@@ -0,0 +1,251 @@
+  ]
+}
+```
+
+### Others
+
+You can use your DNS providers we portal to do similar. You just need to make sure the record is of type "A" and set to your IP
+
+## Applying the YAML in Kubernetes
+
+Now assuming the A Record is set, we can apply first the manifest.yaml:
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should have a language specified

(MD040, fenced-code-language)